recipes/_agent_common_directories.rb

```diff
@@ -11,6 +11,11 @@
       mode '755'
     end
     recursive true
-    notifies :restart, 'service[zabbix_agentd]'
+    if node['platform_family'] == 'windows'
+      notifies :run, 'execute[install_zabbix_agentd]'
+      notifies :run, 'execute[start_zabbix_agentd]'
+    else
+      notifies :restart, 'service[zabbix_agentd]'
```

Comment:
Nope, I did it in the previous pr. We have to install zabbix agent on windows using "execute" (running commands) instead of "service" because the default "windows_service" provider is not working properly in our case